lib/matplotlib/tests/test_text.py

```diff
@@ -709,3 +709,12 @@ def test_update_mutate_input():
     t.update(inp)
     assert inp['fontproperties'] == cache['fontproperties']
     assert inp['bbox'] == cache['bbox']
+
+
+def test_invalid_rotation_values():
+    with pytest.raises(ValueError,
+                       match="'invalid string' is not a valid value for s"):
```

Comment:
```suggestion
                       match="'invalid string' is not a valid value for rotation"):
```
... to fix the test.

Reply:
woops, thanks for catching that...